tests/testthat/test-read_xml_safely.R

```diff
@@ -0,0 +1,18 @@
+context("Safe reading of XMLs")
+
+inputs <- c(
+  "<p>Some paragraph with <b>text in bold</b> which should be OK</p>",
+  "<p>Some paragraph with <b>also in bold</b> with a an illegal character \u0019</p>"
+)
+
+test_that("proper XML is parsed correctly", {
+  expect_equal( read_xml_safely(inputs[1]), xml2::read_xml(inputs[1]) )
```

Comment:
@mbojan  The Travis errors occur here and the test at line 15.

Looks like the code being run runs fine, but the error happens in the testthat call. The error message for both is 

> Error in unclass(target) : cannot unclass an external pointer

Perhaps it's having trouble comparing `xml_document` objects - perhaps test for some aspect of the parsed object that is just text